research/4.x/alerts.md

```diff
@@ -0,0 +1,146 @@
+--
+parent: Research
+
+--
+# Investigating OpenShift Alerts 
+
+*Note: this is an article about a research activity on how users interact with Alertmanager alerts within OpenShift. If you're looking for documentation about Alertmanager alerts, please go to [this link.](https://prometheus.io/docs/alerting/alertmanager/)*
+
+## Introduction
+
+Monitoring is a critical aspect of architecting and maintaining a successful OpenShift environment. One of the most common ways a user interacts with monitoring is through an alert, which can help direct the user's attention to the problem and, hopefully, how to resolve it. Our user experience design research team conducted a mixed-methods, online study with 89 internal OpenShift users and 14 OpenShift customers to explore (a) **what makes an alert useful** and (b) **what a user typically does after seeing an alert**. This information helped evaluate the current state of OpenShift alerts and guide what design changes in the web console could better support our users.
+
+## Research Plan
+
+ We found 11 especially frequent alerts by querying existing engineering data on active OpenShift clusters. These alerts served as the examples we used across our study. 
+
+We used an online survey platform, *Qualtrics*, to implement our study. The survey began with background questions about the participants' experience levels and typical usage patterns. After, we displayed 5 of 11 alerts (randomly) to each user. After viewing an alert and its accompanying alert description, we asked users how confident they felt in their ability to resolve the alert on a scale from 1-7, 1 being the least confidence and 7 being the most confident. We also allowed them to write in why they gave the numerical score they chose.
+
+Unlike many applications and software, OpenShift is often deployed 'on-premise' so that standard product analytics are not possible to collect. This poses a unique challenge to assessing user behavior in OpenShift. To get around this, we implemented a [first-click test](https://www.usability.gov/how-to-and-tools/methods/first-click-testing.html) where users clicked on a web console overview dashboard screenshot. This allowed us to have some behavioral assessment on their strategy for resolving each alert. This first-click component occurred after each confidence rating with the same alert and allowed for an open response to describe why they clicked the area they did. 
+
+![](https://github.com/carljpearson/openshift_alerting/blob/master/img/1-KubePodNotReady.png?raw=true)
```

Comment:
Could you update this PR to include an `/img` folder so that the images are stored directly in the repo? I worry about external image references eventually breaking.

Syntax:
```
    ![](img/notifications-icon.png)
```